scipy/stats/tests/test_multivariate.py

```diff
@@ -1635,6 +1637,189 @@ def test_haar(self):
         res = kstest(x.ravel(), uniform(-np.pi, 2*np.pi).cdf)
         assert_(res.pvalue > 0.05)
 
+
+class TestMultivariateHypergeom(object):
+    def test_logpmf(self):
+        vals1 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=7)
```

Comment:
```suggestion
        # Ground truth value from R dmvhyper
        vals1 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=7)
```

Reply:
I used `extraDistr` package's `dmvhyper` function to evaluate the pmf. I have not used the anaconda distribution but am pretty sure it can be installed using `install.packages('extraDistr', dependencies = TRUE)` (in windows, [Rtools](https://cran.r-project.org/bin/windows/Rtools/) is required for cpp compilation. I don't know if this is included in anaconda). Here is the documentation of [dmvhyper](https://www.rdocumentation.org/packages/extraDistr/versions/1.9.1/topics/MultiHypergeometric) function in `extraDistr`.

Once you have `extraDistr`, `dmvhyper` accepts the same arguments as this distribution's pmf. For example:

```r
x = c(3, 4)
m = c(5, 10)
n = 7
dmvhyper(x, m, n, log=TRUE)
```

Reply:
Thanks!  I didn't figure out what library `dmvhyper` was in, but it turns out that [rrdio](https://rdrr.io/snippets/) has `extraDistr` already so it's really easy to run this.
One thought for the future is to use e.g. `options(digits=10)` - enough digits so that you don't have to specify tolerances on `allclose`. Don't need to change it IMO, though.

Reply:
Thanks, I checked all these.